cabal.project

```diff
@@ -54,305 +68,82 @@ package cryptonite
-
-package cardano-tracer
-  tests: True
-
-package trace-resources
-  tests: True
-
-package locli
-  tests: True
-
--- The following is needed because Nix is doing something crazy.
-package byron-spec-ledger
-  tests: False
-
-package vector-map
-  ghc-options: -Werror
-
-package iohk-monitoring
-  tests: False
-
-package ouroboros-consensus-test
-  tests: False
-
-package ouroboros-consensus-cardano-test
-  tests: False
-
-package ouroboros-network
-  tests: False
-
-package ouroboros-network-framework
-  tests: False
-
-package plutus-tx
-  tests: False
-
-package prettyprinter-configurable
-  tests: False
-
-package small-steps
-  tests: False
-
-package small-steps-test
-  tests: False
-
-package tx-generator
-  tests: True
+constraints:
+    hedgehog >= 1.0
+  , bimap >= 0.4.0
+  , libsystemd-journal >= 1.4.4
+  , systemd >= 2.3.0
+    -- systemd-2.3.0 requires at least network 3.1.1.0 but it doesn't declare
+    -- that dependency
+  , network >= 3.1.1.0
+  , HSOpenSSL >= 0.11.7.2
+  , algebraic-graphs < 0.7
+  , protolude < 0.3.1
+    -- TODO: these should be set in cabal files, but avoiding setting them in lower dependencies for initial CHaP release
+  , cardano-prelude == 0.1.0.0
+  , base-deriving-via == 0.1.0.0
+  , cardano-binary  == 1.5.0
+  , cardano-binary-test == 1.3.0
+  , cardano-crypto-class  == 2.0.0
+  , cardano-crypto-praos  == 2.0.0
+  , cardano-crypto-tests  == 2.0.0
+  , cardano-slotting  == 0.1.0.0
+  , measures == 0.1.0.0
+  , orphans-deriving-via == 0.1.0.0
+  , strict-containers == 0.1.0.0
+  , plutus-core == 1.0.0.0
+  , plutus-ledger-api == 1.0.0.0
+  , plutus-tx == 1.0.0.0
+  , plutus-tx-plugin == 1.0.0.0
+  , prettyprinter-configurable == 0.1.0.0
+  , plutus-ghc-stub == 8.6.5
+  , word-array == 0.1.0.0
 
-package goblins
-  tests: False
+package snap-server
+  flags: +openssl
 
-package io-classes
-  tests: False
+package comonad
+  flags: -test-doctests
 
-package cardano-ledger-alonzo-test
-  tests: False
+allow-newer:
+  *:aeson,
+  monoidal-containers:aeson,
+  size-based:template-haskell,
+  snap-server:attoparsec,
 
--- ---------------------------------------------------------
+-- IMPORTANT
+-- Do NOT add more source-repository-package stanzas here unless they are strictly
+-- temporary! Please read the section in CONTRIBUTING about updating dependencies.
 
--- The two following one-liners will cut off / restore the remainder of this file (for nix-shell users):
--- when using the "cabal" wrapper script provided by nix-shell.
--- --------------------------- 8< --------------------------
--- Please do not put any `source-repository-package` clause above this line.
+-- TODO (mpj): remove this once https://github.com/input-output-hk/ekg-forward/pull/10
+-- is merged and released to CHaP
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/ekg-forward
+  tag: 297cd9db5074339a2fb2e5ae7d0780debb670c63
+  --sha256: 1zcwry3y5rmd9lgxy89wsb3k4kpffqji35dc7ghzbz603y1gy24g
```

Comment:
This has been resolved already as per https://github.com/input-output-hk/cardano-haskell-packages/pull/39?

Reply:
That hash is indeed ekg-forward-0.1.0 in CHaP

Reply:
Fixed in [dff694c](https://github.com/input-output-hk/cardano-node/pull/4540/commits/dff694cae80ec2dd8cbec66b19fc82f92236a574)